larray/inout/stata.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import absolute_import, print_function
+
+import pandas as pd
+
+from larray.inout.pandas import from_frame
+
+__all__ = ['read_stata']
+
+
+def read_stata(filepath_or_buffer, index_col=None, sort_rows=False, sort_columns=False,
+               **kwargs):
+    """
```

Comment:
What about implementing ``StataHandler`` class and ``Session.to_stata`` ?

Reply:
Sure but I would consider that very low priority. I mean, it is indeed theoretically useful, but we have no demand for that yet, nor the intuition that it would help our current users streamline their work.

Reply:
Totally agree. I was asking the question just in case.